scripts/measure-performance.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+set -eu
+
+[ "$#" -lt 1 -o "$(($# % 2))" != "0" ] && {
+    echo >&2 "Usage: $(basename "$0") name1 URL1 [name2 URL2]..."
+    exit 1
+}
+
+if ! which jq >/dev/null; then
+    echo >&2 "Error: jq not found in PATH. You should install it first."
+    exit 1
+fi
+
+REFERENCE_URL="https://app.gatesolve.com/"
+
+BINDIR="$(dirname "$0")"
+REPOROOT="$(git -C "$BINDIR" rev-parse --show-toplevel)"
+
+RELATIVEDIR="performance-results/$(date --utc '+%Y%m%dT%H%M%SZ')"
+ABSOLUTEDIR="$REPOROOT/$RELATIVEDIR"
+mkdir -p "$REPOROOT/performance-results"
+mkdir "$ABSOLUTEDIR"
+
+run_measurement() {
+    name="$1"
+    url="$2"
+    # pwmetrics cannot deal with an absolute path but needs RELATIVEDIR instead
+    yarn --cwd "$REPOROOT" run pwmetrics --runs 2 "$url" --json --output-path "$RELATIVEDIR/$name".json
```

Comment:
We forgot to change `--runs` back to 30.

Reply:
Good catch!